### DISCUSSION TITLE
Failed Plugins

### DISCUSSION THREAD

**Discussion by aluntay** [apohl79/audiogridder#846] [Help]
Hi, I'm getting lots of failed Acustica Audio Plugins on my PC Slave, I've tried removing them from the blacklist, using the latest beta version of AG (originally used the stable version) reinstalling the plugins, installing VST3 versions (originally used VST2) but it keeps happening. Any suggestions please?

Edit: for clarity, around half my Acustica Plugins load correctly, the other half don't and it seems random? And also, the Audio Unit versions on my Mac load fine using AudioGridder on my Mac.

Mac Pro Early 2009 2 x 3.46 GHz 6-Core Intel Xeon Mojave 10.14.6

PC Intel(R) Core(TM) i7-4790 CPU @ 3.60GHz   3.60 GHz Widnows 10 Home

![Screenshot 2021-12-12 at 09 53 29](https://user-images.githubusercontent.com/96002239/145707897-6c4a4ff1-dff1-4a18-b597-a11944ff2457.png)

![Screenshot 2021-12-12 at 17 53 02](https://user-images.githubusercontent.com/96002239/145723665-b7a668f6-e440-4847-9e8a-b8ea8fea1cca.png)

**Comment by Henri75** [comment]
I have similar issues.
It is most common with AA plugins, but happens less often with others too.

**Comment by aluntay** [comment]
On the suggestio of AA I change the file path to not have any spaces and it worked.

**Comment by lorenzolamasse** [comment]
I have all Bluecat plugins failing as well, I have tried to rename them to remove any space, and even put them in c:\vst3 to avoid any space in pathname as well, to no avail, they all fail, joining trace file here in case you can have a look...
[AudioGridderServer_2022-02-18_08-02-49.zip](https://github.com/apohl79/audiogridder/files/8094842/AudioGridderServer_2022-02-18_08-02-49.zip)
.

**Comment by macmicke-se** [comment]
I'm seeing this with FXpansions (intel) plug-ins using 1.2.0b6 intel in macOS 11.6.3 on M1. There are certainly no spaces in my paths to the plug-ins.
